internal/distro/rhel9/distro.go

```diff
@@ -126,7 +126,7 @@ func (d *distribution) getDefaultImageConfig() *distro.ImageConfig {
 
 func New() distro.Distro {
 	// default minor: create default minor version (current GA) and rename it
-	d := newDistro("rhel", 0)
```

Comment:
Same thing as before, another sidenote; can we document somewhere that these values 'need to be changed' every new RHEL minor?

Reply:
Yes, I want to figure out a better way to do this too.  I think it should be defined in the distro or the distro registry, but not anywhere else (like, I wouldn't delegate it to an external configuration).  But I think it should be somewhere where it's clearly visible, easy to change, and we should have rules for when to update it.  Perhaps some release automation/test to avoid releasing composer with, for example, RHEL 8.9 support in RHEL 8.9 but where `rhel-8 == rhel-8.8`.